css/styles.css

```diff
@@ -499,220 +454,3 @@ textarea {
-  }
-
-  #page-head {
-    background: transparent;
-    justify-content: center;
-  }
-
-  #logo {
-    position: absolute;
-    display: none;
-  }
-
-  .fa-bars {
-    position: absolute;
-    display: none;
-  }
-
-  #menu-items {
-    display: flex;
-    padding-right: 40px;
-    list-style-type: none;
-    width: 100%;
-  }
-
-  #menu-items a {
-    margin-left: 2rem;
-    color: #42526e;
-    font-weight: 600;
-    text-decoration: none;
-    font-size: 15px;
-  }
-
-  /* headline section */
-
-  #headline {
-    padding-bottom: 1rem;
-    background: transparent;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-  }
-
-  #introduction {
-    margin: 100px 260px 0 260px;
-  }
-
-  #introduction h1 {
-    font-size: 48px;
-    line-height: 1.1em;
-    font-weight: 800;
-  }
-
-  #first-break {
-    display: none;
-  }
-
-  #text-section p {
-    font-size: 20px;
-    line-height: 1.2em;
-    width: 800px;
-    padding-bottom: 2rem;
-  }
-
-  /* porjects section */
-
-  #rule {
-    display: none;
-  }
-
-  #break {
-    display: none;
-  }
-
-  #projects-container {
-    grid-template-columns: 1fr 1fr 1fr;
-    margin: 0 12rem;
-    padding-bottom: 5rem;
-  }
-
-  .project-file {
-    margin-bottom: 2rem;
-  }
-
-  /* about section */
-  #about {
-    position: relative;
-    padding-bottom: 7rem;
-  }
-
-  #about-background-images {
-    position: absolute;
-    top: 0;
-    display: block;
-    width: 100%;
-    z-index: -1;
-  }
-
-  #about-background-image-1 {
-    float: left;
-  }
-
-  #about-background-image-2 {
-    float: right;
-  }
-
-  #about-text {
-    width: 920px;
-    margin: 0 auto;
-    font-size: 20px;
-    line-height: 32px;
-  }
-
-  #stack-list {
-    grid-template-columns: 1fr 1fr 1fr;
-    margin: 2rem 12rem;
-  }
-
-  /* contact section */
-
-  #background-img {
-    display: none;
-  }
-
-  #contact-desktop-bg {
-    position: absolute;
-    bottom: 19%;
-    left: 20%;
-    display: block;
-  }
-
-  #contact-bg-left {
-    position: absolute;
-    bottom: 19%;
-    left: 0;
-    display: block;
-  }
-
-  #contact-bg-rightt {
-    position: absolute;
-    bottom: 19%;
-    right: 0;
-    display: block;
-  }
-
-  #top {
-    display: grid;
-    grid-template-columns: 1fr 1fr;
-    margin: 0 14rem;
-    margin-bottom: 10rem;
-  }
-
-  #contact-section h3 {
-    font-size: 30px;
-    line-height: 52px;
-    width: 100%;
-    text-align: left;
-    margin-top: 2rem;
-    font-weight: 700;
-    padding-right: 1rem;
-  }
-
-  fieldset {
-    padding: 0;
-  }
-
-  #message {
-    border: solid 1px #36b37e;
-  }
-
-  #contact-section button {
-    margin-top: 2rem;
-  }
-
-  #contact-email {
-    display: none;
-  }
-
-  #social-media-links {
-    display: flex;
-    justify-content: center;
-    bottom: 0;
-    left: 45%;
-  }
-
-  #contact-links2 {
-    position: absolute;
-    bottom: 2%;
-    border-top: 1px solid rgb(158, 158, 158);
-    width: 100%;
-    padding-top: 1rem;
```

Comment:
[CAUTION] IDs have a much higher specificity than classes. A class on its own can't overwrite styles belonging to an ID. To  beat the ID, you would need either more IDs or to use ! important in your css which can also lead to specifity wars in your stylesheets. My advise is kindly use classes not IDs to avoid specifity problems. 😃